### PR TITLE
Update tlg2200.tlg00453._cts_.xml

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Edited language tagging
Edited work title
Added in edition metadata

### CODE REVIEW

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/997)*

File: data/tlg2200/tlg00453/__cts__.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg2200" xml:lang="grc" urn="urn:cts:greekLit:tlg2200.tlg00453">
-  <ti:title xml:lang="eng">Orationes 53</ti:title>
+  <ti:title xml:lang="lat">Oration 53</ti:title>
```
Comment (sonofmun) [inline data/tlg2200/tlg00453/__cts__.xml]: Should this be "Oratio" instead of "Oration"?

Comment (AlisonBabeu) [inline data/tlg2200/tlg00453/__cts__.xml]: Sigh, yes, yes it should. So many Oratios so little time.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/1113)*

**Pull request merged**